courseware/api.py

```diff
@@ -52,7 +52,7 @@
 OPENEDX_REGISTER_USER_PATH = "/user_api/v1/account/registration/"
 OPENEDX_REQUEST_DEFAULTS = dict(country="US", honor_code=True)  # noqa: C408
 
-OPENEDX_SOCIAL_LOGIN_XPRO_PATH = "/auth/login/mitxpro-oauth2/?auth_entry=login"
```

Comment:
@arslanashraf7 can we take the value for this from .env? This way we can have different values for production and rc and marketing app wont be freeze until the edx sumac production release?

Reply:
@arslanashraf7 I have pushed the changes in the PR and created https://github.com/mitodl/ol-infrastructure/pull/2824